draft-ietf-capport-architecture.xml

```diff
@@ -174,17 +170,17 @@
             A device MAY query this API at any time to determine whether the
             network is holding the device in a captive state.
          </t>
-         <t>End-user devices are notified of captivity with Captive Portal Signals
-            in response to traffic. This notification should  work with any Internet protocol, not just
-            clear-text HTTP. This notification does not carry the portal URI;
-            rather it provides a notification to the User Equipment that it
-            is in a captive state. This document will specify requirements for the signaling
-        protocol which will generate Captive Portal Signals.
+         <t>End-user devices may be notified of captivity with Captive Portal Signals
```

Comment:
Since `may` can be confused with `MAY`, I sometime suggest using `can` for this purpose, when you are just stating a fact.